frog/imports/client/SingleActivity/Finish.js

```diff
@@ -40,14 +39,20 @@ function Finish(
       </IconButton>
       <Typography variant="h3" component="h2" className={classes.padded_text}>
         You're all set, please share this link with the participants:{' '}
-        {BASE_URL + '/' + url.public}
+        {BASE_URL + '/' + slug}
       </Typography>
       <Typography variant="h4" component="h3" className={classes.padded_text}>
         Here's a secret link to your behind-the-scene view of the dashboard:{' '}
-        <Link to={'/' + url.dashboard + '?u=' + Meteor.userId()}>
-          {BASE_URL + '/' + url.dashboard + '?u=' + Meteor.userId()}
+        <Link to={'/t/' + slug + '?u=' + Meteor.userId()}>
+          {BASE_URL + '/t/' + slug + '?u=' + Meteor.userId()}
```

Comment:
We should move `/t` to a constant to simplify future work. A change to the path could break this link.

Reply:
Let's add another task to think about constants and URLs. Where should those constants be defined and exported from?